Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.psm1

```diff
@@ -26,15 +26,18 @@ function Get-ModifiedFiles {
             }
             Invoke-CommandWithRetry -ScriptBlock { RunAndCheck git fetch origin $baselineSHA | Out-Host }
         }
-        else {
+        elseif ($baselineSHA) {
             $headSHA = git rev-parse HEAD
             Write-Host "Current HEAD is $headSHA"
             Invoke-CommandWithRetry -ScriptBlock { RunAndCheck git fetch origin $baselineSHA | Out-Host }
             Write-Host "Not a pull request, using baseline SHA $baselineSHA and current HEAD $headSHA"
         }
+        else {
+            return $true, @()
```

Comment:
When is it in this case? `$baselineSHA` is a mandatory parameter.

Also, doc should be updated.

Reply:
Removed this. It came from DetermineProjectsToBuild.Action.ps1, where the function was called even if no baseline SHA was found. Probably this lead to the error you have seen - now I added a notice and do not call get-modifiedFiles